requirements.txt

```diff
@@ -1,15 +1,12 @@
-braintree==3.57.1
 frappe
-gocardless-pro==1.11.0
-googlemaps==3.1.1
```

Comment:
`googlemaps` is being used in the delivery trip controller of the Stock module. Although `frappe` installs it because of its dependency, it's best to track it here as well. We may not need to pin it here though.

Couldn't find usages of `braintree` or `pycryptodome` so I think dropping them is ok.

Reply:
Alright, I added googlemaps back in